tests/xammac_tests/CoreWlan/CWKeychainTests.cs

```diff
@@ -0,0 +1,85 @@
+//
+// Unit tests for CVPixelBuffer
+//
+// Authors:
+//	Rolf Bjarne Kvinge <rolf@xamarin.com>
+//
+// Copyright 2016 Xamarin Inc. All rights reserved.
+//
+
+#if !__WATCHOS__
+
+using System;
+using System.Drawing;
+using System.Runtime.InteropServices;
+using Foundation;
+using ObjCRuntime;
+using CoreWlan;
+using Security;
+
+using NUnit.Framework;
+using MonoTouchFixtures.Security;
+
+namespace MonoTouchFixtures.CoreWlan {
+
+	[TestFixture]
+	[Preserve (AllMembers = true)]
+	public class CWKeychainTests {
+
+		CWKeychainDomain domain;
+		NSData ssid;
+
+		[SetUp]
+		public void SetUp ()
+		{
+			domain = CWKeychainDomain.System;
```

Comment:
Will this test add stuff to my system keychain? Can we make sure the test cleans up whatever it adds if that's the case?

Reply:
it won't AFAIK.